pontoon/sync/utils.py

```diff
@@ -118,3 +126,42 @@ def import_uploaded_file(
         return badge_name, badge_level
     else:
         raise Exception("Upload failed.")
+
+
+android_nl = compile(r"\s*\n\s*")
+android_esc = compile(r"(?<!\\)\\([nt])\s*")
+
+
+def parse_pontoon_message(
+    db_format: str, string: str, meta: list[list[str]]
+) -> Message:
+    """
+    Parse an Entity.string or a Translation.string as a moz.l10n Message.
+
+    This function should be removed once the string message representations are removed.
+    """
```

Comment:
This function is currently only used by the base/0096 migration, but it'll be needed later for translations as well.

Reply:
```suggestion
def parse_pontoon_message(
    db_format: str, string: str, meta: list[list[str]]
) -> Message:
    """
    Parse an Entity.string or a Translation.string as a moz.l10n Message.

    TODO: This function should be removed once the string message representations are removed.
    """
```